File: javascript/1043.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var values = input.split(' ');
var a = parseFloat(values[0]);
var b = parseFloat(values[1]);
var c = parseFloat(values[2]);
function check(v1, v2, v3) {
     var calc = v1 + v2;
     if(calc > v3) {
          return true
     } else {
          return false
     }
}

var abc = check(a, b, c);
var acb = check(a, c, b);
var bca = check(b, c, a);
var area = ((a + b) * c) / 2;
if(abc && acb && bca) {
     console.log('Perimetro = ' + (a+b+c).toFixed(1));
} else {
     console.log('Area = ' + area.toFixed(1));
}

File: C++/1043.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int check(float v1,float v2, float v3){
    float calc;
    calc=v1+v2;
    
    if(calc>v3){
        return 1;
    }else{
        return 0;
    }
}

int main(){
    
    float a,b,c,sum,area;
    cin >> a >> b >> c;
    sum=a+b+c;
    area=((a+b)*c)/2.0;
    int abc,acb,bca;
    abc=check(a,b,c);
    acb=check(a,c,b);
    bca=check(b,c,a);
    if(abc == 1 && acb == 1 && bca == 1){
        cout << "Perimetro = " << fixed << setprecision(1) << sum << endl;
    }else{
        cout << "Area = " << fixed << setprecision(1) << area << endl;
    }
    
    return 0;
}